src/Init.php

```diff
@@ -288,6 +292,12 @@ function feature($name)
 	'import'  => $di->lazyNew('Ushahidi\Core\Usecase\ImportUsecase'),
 ];
 
+// Add custom usecases for posts_bulk
+$di->params['Ushahidi\Factory\UsecaseFactory']['map']['posts_bulk'] = [
```

Comment:
I'd be inclined to keep the `resource` in the controller as `posts` but use `bulk_update` and `bulk_delete` as the 'actions` here .. that makes more semantic sense. And I think it should just work.